src/main.scss

```
@@ -0,0 +1,40 @@
+body {
+  margin: 0;
+  font-family: Arial;
+  font-size: 100px;
+  color: white;
+}
+
+.container {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr;
```

как упростить себе жизнь если сегодня 3 колонки, завтра 10, послезавтра 40?
легко!
```suggestion
  grid-template-columns: repeat(3, 1fr);
```